### PR TITLE
OCTO-434: Create Helm chart for kafka-health service

### PR DESCRIPTION
**Pull request by tyrel**
Add the kafka-health service for Octocat. This is a copy of the sink service with the names changed.

### CODE REVIEW

File: charts/gp-reporting-kafka-health/v0.0.1/questions.yml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  ####
+  - variable: build.commitSHA
+    type: string
+    required: true
+    group: "Application"
+    label: Application Build SHA
+    description: The commit SHA of the GitPrime Application build you want to deploy.
+    default: latest
+
+  - variable: volumes.enableLocalMount
```
Comment (mxsilverstein) [inline charts/gp-reporting-kafka-health/v0.0.1/questions.yml]: For this app you can probably skip the volumes stuff one if there's no local storage.

File: charts/gp-reporting-kafka-health/v0.0.1/questions.yml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  # Reporting Kafka Health Service Settings
+  - variable: application.webPort
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Application port binding
+    description: The port binding that the application uses to listen to web requests
+    default: "8080"
+
+  - variable: application.nodePort
```
Comment (mxsilverstein) [inline charts/gp-reporting-kafka-health/v0.0.1/questions.yml:72]: Unfortunately there are a finite set of nodeports and "30680" is already taken by data sink. "30780" is free. Let's use that. This won't be an issue once we have a load balancer, which @cflanny is working on, and we can stop using nodeports.

File: charts/gp-reporting-kafka-health/v0.0.1/questions.yml [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    description: The Java options for the services.
+    default: "-d64 -Xms2g -Xmx4g"
+
+  - variable: application.resources.requests.cpu
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Reporting Kafka Health Service Requested CPU
+    description: The number of CPUs the services will request.
+    default: "2"
```
Comment (mxsilverstein) [inline charts/gp-reporting-kafka-health/v0.0.1/questions.yml]: This is requesting 2 CPUs, which is probably more than this small app needs. This is going to be a low CPU app so I'd set this to 1/10 of a CPU, or "100m". More info about resources can be found here: https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#specify-a-cpu-request-and-a-cpu-limit

File: charts/gp-reporting-kafka-health/v0.0.1/questions.yml (hunk 1 above)
Comment (tyrel) [reply charts/gp-reporting-kafka-health/v0.0.1/questions.yml]: Removed all volumes configuration

File: charts/gp-reporting-kafka-health/v0.0.1/questions.yml (hunk 2 above)
Comment (tyrel) [reply charts/gp-reporting-kafka-health/v0.0.1/questions.yml:72]: Changed ports (in 2 files) to 30780

File: charts/gp-reporting-kafka-health/v0.0.1/questions.yml (hunk 3 above)
Comment (tyrel) [reply charts/gp-reporting-kafka-health/v0.0.1/questions.yml]: Set to 100 milliCPUs

File: charts/gp-reporting-kafka-health/v0.0.1/questions.yml [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    description: The maximum amount of memory a worker will request.
+    default: "4Gi"
+
+  - variable: application.resources.limits.cpu
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Reporting Kafka Health Service CPU Limit
+    description: The maximum number of CPUs the services can use.
+    default: "4"
```
Comment (mxsilverstein) [inline charts/gp-reporting-kafka-health/v0.0.1/questions.yml]: This should probably be lower. 1 Should be fine.

**Review by mxsilverstein — APPROVED** [review]
LGTM with the the one last minor update (CPU limit) - approved to merge at will.

File: charts/gp-reporting-kafka-health/v0.0.1/questions.yml (hunk 4 above)
Comment (tyrel) [reply charts/gp-reporting-kafka-health/v0.0.1/questions.yml]: Done

**Pull request merged**